data/net/minecraft/core/Cursor3D.mapping

```diff
@@ -1 +1,15 @@
-CLASS net/minecraft/core/Cursor3D
+CLASS net/minecraft/core/Cursor3D net/minecraft/core/Cursor3D
+	COMMENT This class represents a cuboid. You can iterate over it using the methods {@link Cursor3D#advance()} and {@link Cursor3D#nextX()}, {@link Cursor3D#nextY()}, {@link Cursor3D#nextZ()}
+	METHOD <init> (IIIIII)V
+		ARG 1 originX
+		ARG 2 originY
+		ARG 3 originZ
+		ARG 4 x2
+		ARG 5 y2
+		ARG 6 z2
+	METHOD getNextType ()I
+		COMMENT Returns the Type of the next BlockPos.
```

Comment:
Few things...
1) why is `Type` in caps, it isn't a class.
2) What does BlockPos have to do with this? 
The whole class works with ints and is keeping track of the individual int, yes they are being put into a BlockPos later down the line, but you are not getting a "type" of a BlockPos or even strictly working with BlockPos at all.
3) This comment is incredibly misleading, BlockPos don't have a Type.